build.gradle.kts

```diff
@@ -1,10 +1,13 @@
+import org.gradle.internal.impldep.org.junit.platform.launcher.EngineFilter.includeEngines
```

Comment:
Move under file copyright comment block

Reply:
Hmm. I'd rather not because IntelliJ automatically adds and re-orders these import statements. In our ZAC project we have the copyright under the import statements for the same reason.